mypyc/test-data/irbuild-basic.test

```diff
@@ -1036,15 +1036,16 @@ L0:
 
 [case testLoadComplex]
 def load() -> complex:
-    return 5j+1.0
+    real = 1
+    return 5j+real
```

Comment:
The intent of this change is to preserve the old test case behavior where constant folding wasn't performed.